jetty-websocket/websocket-jetty-api/src/main/java/org/eclipse/jetty/websocket/api/extensions/ExtensionConfig.java

```diff
@@ -34,7 +34,7 @@
 
     private static ExtensionConfig.Parser getParser()
     {
-        return ServiceLoader.load(ExtensionConfig.Parser.class).findFirst().get();
+        return ServiceLoader.load(ExtensionConfig.Parser.class).iterator().next();
```

Comment:
Why isn't this using the TypeUtil method to skip exceptions?

Reply:
websocket-api does not depend on jetty-util (or any other jetty dependency for that matter)